pixi.lock

```diff
@@ -8458,28 +8458,6 @@ packages:
   license_family: BSD
   size: 15322
   timestamp: 1712542844430
-- kind: conda
```

Comment:
Why are there changes to this file?

Reply:
`pixi.lock` file is usually auto updated. Every one runs `pixi run ....`, pixi will check if our dependency has update and then update the lock file.